portable_mds/tests/test_mds.py

```diff
@@ -2,25 +2,25 @@
 import uuid
 import pytest
 from portable_mds.template.core import NoEventDescriptors, NoRunStart
-from metadatastore.test.test_mds import (
-    test_bad_bulk_insert_event_data,
-    test_bad_bulk_insert_event_timestamp,
-    test_bad_event_desc,
-    test_bulk_insert,
-    test_iterative_insert,
-    test_bulk_table,
-    test_cache_clear_lookups,
-    test_custom_warn,
-    test_double_run_stop,
-    test_event_descriptor_insertion,
-    # test_fail_runstart,
-    test_find_run_start,
-    test_find_run_stop,
-    test_insert_run_start,
-    # test_no_evdesc,
-    test_pickle,
-    test_run_stop_by_run_start,
-    test_run_stop_insertion)
+# from metadatastore.test.test_mds import (
```

Comment:
The goal of the fixture here is to make sure portable_mds passes all the metadatastore tests w/o having to rewrite them. mdservice can even use portable_mds as its backend (I tested this, it does 👏 ) .